tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AzureFunctionsTests.cs

```diff
@@ -229,6 +250,80 @@ public async Task SubmitsTraces()
                 using var s = new AssertionScope();
 
                 await AssertIsolatedSpans(spans);
+
+                spans.Count.Should().Be(expectedSpanCount);
+            }
+        }
+    }
+
+    [UsesVerify]
+    [Collection(nameof(AzureFunctionsTestsCollection))]
+    public class IsolatedRuntimeV4AspNetCore : AzureFunctionsTests
```

Comment:
We're currently only building this sample for .NET 8 and .NET 9, so we'll need to add `#if` around these tests if that's correct - otherwise we should "just" need to build for more tfms.

Reply:
Fixed this, added the missing TFMs